### ISSUE TITLE
v0.1.6 fails with `ReferenceError: boolean is not defined`

### ISSUE THREAD

**Issue by kategengler** [Turbo87/auto-dist-tag#22]
Fails with this error:

```
/Users/katie/.config/yarn/global/node_modules/auto-dist-tag/cli.js:17
      type: boolean,
            ^

ReferenceError: boolean is not defined
```

I believe this line https://github.com/Turbo87/auto-dist-tag/blob/master/cli.js#L17 should have `boolean` in quotes.

**Comment by Turbo87** [comment]
obviously I should've tested #21 before merging it... 🙈

**Comment by locks** [comment]
mea culpa 😩

**Comment by Turbo87** [comment]
fix released as v0.1.7
